tests/readers/test_netcdf_reader.py

```diff
@@ -215,6 +216,28 @@ def test_netcdf_proj_string_laea():
     assert r[1] == "+proj=eqc +datum=WGS84 +units=m +no_defs"
 
 
+def test_get_fields_missing_standard_name_attr_in_coord_array():
+    """test _get_fields() can handle a missing 'standard_name' attr in coordinate data arrays"""
+
+    # example dataset
+    fs = from_source("file", earthkit_file("docs/examples/test.nc"))
```

Comment:
I suggest you use`earthkit_examples_file` (from `earthkit.data.testing`) to hide the relative path like this:

`fs = from_source("file", earthkit_examples_file("test.nc"))`

Reply:
This commit implements that recommendation..  https://github.com/ecmwf/earthkit-data/commit/c7d519abca2a1ceba5fec0d0128ced60a1159db2